models/sessions/scratch/databricks/snowplow_web_sessions_this_run.sql

```diff
@@ -287,6 +344,11 @@ select
         {%- for conv_def in var('snowplow__conversion_events') %}
     {{ snowplow_web.get_conversion_columns(conv_def, names_only = true)}}
         {%- endfor %}
+    {% if var('snowplow__total_all_conversions', false) %}
+    ,{%- for conv_def in var('snowplow__conversion_events') %}{{'cv_' ~ conv_def['name'] ~ '_volume'}}{%- if not loop.last %} + {% endif -%}{%- endfor %} as cv__all_volume
```

Comment:
Cast to `int`?

Reply:
The volume is just from a count column, so things should flow through as an int for these regardless